frontend/src/components/productGrid/ProductGridComponent.js

```diff
@@ -48,18 +45,21 @@ function ProductGridComponent({ selectedCategory }) {
     ];
 
     if (productList === [] || productList === undefined || productList === null) {
+        console.error(
+            'Products data not recieved.Please verify the API calls and Backend Connection.'
+        );
         return <LoadingComponent />;
```

Comment:
```suggestion
        return <LoadingComponent loading />;
```

otherwise it will hide itself 😉